packages/categorize/configure/src/design/categories/__tests__/__snapshots__/choice-preview.test.jsx.snap

```diff
@@ -14,6 +14,7 @@ exports[`ChoicePreview snapshot renders 1`] = `
     onRemoveChoice={[Function]}
   >
     <HtmlAndMath
+      className="undefined"
```

Comment:
Is there any issue with the className being undefined?

Reply:
I think no. All tests have passed. I don't know exactly how snapshots should be updated in this case, but we have this in multiple-choice and match-list too